buggy_context_manager.py

```
@@ -0,0 +1,29 @@
+            if getattr(self, 'file', None):
+                self.file.close()
+        except Exception:
+            pass
+        # Do not suppress exceptions; propagate by returning False
+        return False
+
+# Usage example
+try:
+    with FileManager('nonexistent.txt', 'R') as f:
+        print(f.read())
+except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)